bootstrap/aws/util.sh

```diff
@@ -84,3 +85,7 @@ ovpn_client_config() {
     done
 }
 
+get_network_identifier() {
```

Comment:
Just minor thing: If we pass the output of $( terraform output vpc_cidr_block.ip  ) as an argument to get_network_identifier the we can test that function in test.sh